x/ccv/provider/keeper/proposal_test.go

```diff
@@ -1002,13 +1002,8 @@ func TestBeginBlockInit(t *testing.T) {
 	}
 
 	// opt in a sample validator so the chain's proposal can successfully execute
-	providerKeeper.SetOptedIn(ctx, pendingProps[5].ChainId, providertypes.OptedInValidator{
```

Comment:
Not related to this PR but this was making the `proposal_test` fail: `SetOptedIn` does not take an `OptedInValidator` but a `ProviderConsAddress`.